README.md

```diff
@@ -85,932 +119,667 @@ Unlike many authorization systems where a service controls access to resources i
 
-If an attenuated resource or capability is desired, it MUST be explicitly listed without the `ucan` URI scheme.
+### `/` AKA "Top"
 
-``` js
-{ 
-  "ucan:bafkreihogico5an3e2xy3fykalfwxxry7itbhfcgq6f47sif6d7w6uk2ze": {"ucan/*": [{}]}, 
-  "ucan:*": {"ucan/*": [{}]}
-}
-```
+_"Top" (`/`) is the most powerful ability, and as such it SHOULD be handled with care and used sparingly._
 
-## 5.2 Top
+The "top" (or "any", or "wildcard") ability MUST be denoted `/`. This can be thought of as something akin to a super user permission in RBAC. 
 
-The "top" (or "super user") ability MUST be denoted `*`. The top ability grants access to all other capabilities for the specified resource, across all possible namespaces. Top corresponds to an "all" matcher, whereas [delegation] corresponds to "any" in the UCAN chain. The top ability is useful when "linking" agents by delegating all access to resource(s). This is the most powerful ability, and as such it SHOULD be handled with care.
+The wildcard ability grants access to all other capabilities for the specified resource, across all possible namespaces. The wildcard ability is useful when "linking" agents by delegating all access to another device controlled by the same user, and that should behave as the same agent. It is extremely powerful, and should be used with care. Among other things, it permits the delegate to update a Subject's mutable DID document (change their private keys), revoke UCAN delegations, and use any resources delegated to the Subject by others.
 
 ``` mermaid
 %%{ init: { 'flowchart': { 'curve': 'linear' } } }%%
 
 flowchart BT
-  *
+  /
 
-  msg/* --> *
+  /msg --> /
   subgraph msgGraph [ ]
-    msg/send --> msg/*
-    msg/receive --> msg/*
+    /msg/send --> /msg
+    /msg/receive --> /msg
   end
 
-  crud/* --> *
+  /crud --> /
   subgraph crudGraph [ ]
-    crud/read --> crud/*
-    crud/mutate --> crud/*
+    /crud/read --> /crud
+    /crud/mutate --> /crud
 
     subgraph mutationGraph [ ]
-        crud/create --> crud/mutate
-        crud/update --> crud/mutate
-        crud/destroy --> crud/mutate
+        /crud/mutate/create --> /crud/mutate
+        /crud/mutate/update --> /crud/mutate
+        /crud/mutate/destroy --> /crud/mutate
     end
   end
 
-  ... --> *
+  ... --> /
 ```
 
+### Reserved Commands
 
-### 5.2.1 Bottom
-
-In concept there is a "bottom" ability ("none" or "void"), but it is not possible to represent in an ability. As it is merely the absence of any ability, it is not possible to construct a capability with a bottom ability.
+#### `/ucan` Namespace
 
-# 6. Validation
+The `/ucan` Command namespace MUST be reserved. This MUST include any ability string matching the regex `^ucan\/.*`. This is important for keeping a space for community-blessed Commands in the future, such as standard library Commands, such as [Revocation].
```

Comment:
```suggestion
The `/ucan` Command namespace MUST be reserved. This MUST include any ability string matching the regex `^ucan(\/(^\m\/)+)+`. This is important for keeping a space for community-blessed Commands in the future, such as standard library Commands, such as [Revocation].
```
I'm no regex wizard but I'm guessing this regex is older than the prohibition on trailing `/`s, which this newer, uglier regex bans.

Also, I couldn't tell if `/ucan` was supposed to be a valid command (or a valid blank-check for future stdlib commands), but if so, that last `+` (1 or more) should be a `*` (0 or more).